packages/sui-mono/src/config.js

```diff
@@ -6,7 +6,8 @@ const CWD = process.cwd()
 const packageFile = getPackageJson(CWD)
 
 const getWorkspaces = workspaces => {
-  if (workspaces.length === 0) return ['Root']
```

Comment:
isn't this gonna break the commit scopes if there as be no change in a workspace?

Reply:
I think that `getWorkspaces` function should always the same, only return a list (array) of valid paths.
I have checked and fixed all side effects.

Reply:
You could check all `getWorkspaces` usages here:
 https://github.com/SUI-Components/sui/search?q=getWorkspaces&type=code